### PR TITLE
Cleanup reload launchers

### PR DESCRIPTION
**Pull request by PabstMirror**
**When merged this pull request will:**
- Cleanup reload launchers

Minor cleanup of params, and adds a fast exit for non supported weapons.

**Todo**: None of the launcher seem to have a good config for reload time and right now it's a 12 second progress bar which make this module pretty hard to use.

```
        magazineReloadTime = 12;
```

I can't really find any good config value for the real reload time (about 3 seconds), it might be linked to the reload animation??

### CODE REVIEW

**Comment by commy2** [comment]
Hand weapons don't use the magazineReloadTime property. Magazines are reloading instantly and only the animation is preventing the player from shooting.
It is used by vehicles mainly (they obviously have no reload animation)
I think the artificial delay from addWeaponItem is unrelated to this value and the delay is some kind of magic number in the engine. Don't know for sure though.

**Comment by PabstMirror** [comment]
@commy2 Should we just hard code it to be 2-3 second then?
The receiver still has to wait for the ammo load.

**Comment by esteldunedain** [comment]
> Should we just hard code it to be 2-3 second then?

:+1:

**Comment by commy2** [comment]
For now. I don't see them fixing this bug anytime soon.

File: addons/reloadlaunchers/functions/fnc_load.sqf [hunk 1]
```diff
@@ -11,18 +10,16 @@
...
-_target = _this select 1;
-_weapon = _this select 2;
-_magazine = _this select 3;
+params ["_unit", "_target", "_weapon", "_magazine"];
+TRACE_4("params",_unit,_target,_weapon,_magazine);
 
-private "_reloadTime";
-_reloadTime = getNumber (configFile >> "CfgWeapons" >> _weapon >> "magazineReloadTime");
+// private _reloadTime = getNumber (configFile >> "CfgWeapons" >> _weapon >> "magazineReloadTime"); //Not a good config value, use a constant for now:
+private _reloadTime = 2.5;
```
Comment (bux578) [inline addons/reloadlaunchers/functions/fnc_load.sqf]: Wouldn't it be better to introduce a new config value?
Maybe some launchers in the CUP or RHS weapon pack need more reload time?

**Comment by PabstMirror** [comment]
Using @bux578 's suggestion, attempts to use `QGVAR(buddyReloadTime)` from weapon's config, else default to 2.5 seconds.

**Pull request merged**